core/arch/arm/plat-sam/matrix.c

```diff
@@ -510,7 +510,7 @@ static unsigned int matrix_read(int base, unsigned int offset)
 void matrix_write_protect_enable(unsigned int matrix_base)
 {
 	matrix_write(matrix_base, MATRIX_WPMR,
-		    (MATRIX_WPMR_WPKEY_PASSWD | MATRIX_WPMR_WPEN_ENABLE));
+		     (MATRIX_WPMR_WPKEY_PASSWD | MATRIX_WPMR_WPEN_ENABLE));
```

Comment:
my 2 cents: parentheses not needed.

Reply:
Hi @etienne-lms,
Updated by removing the parentheses, thank you.